.env-devel

```diff
@@ -60,8 +60,10 @@ WEBSERVER_RESOURCES_DELETION_TIMEOUT_SECONDS=900
 WEBSERVER_GARBAGE_COLLECTION_INTERVAL_SECONDS=30
 WEBSERVER_MANUAL_MAIN_URL=http://docs.osparc.io/
 WEBSERVER_MANUAL_EXTRA_URL=https://itisfoundation.github.io/osparc-manual-z43/
-WEBSERVER_FOGBUGZ_URL=htpps://z43.fogbugz.com
-WEBSERVER_FOGBUGZ_PROJECT_ID=45
+WEBSERVER_FOGBUGZ_LOGIN_URL=https://z43.fogbugz.com/login
+WEBSERVER_FOGBUGZ_NEWCASE_URL=https://z43.fogbugz.com/f/cases/new?command=new&pg=pgEditBug&ixProject=45&ixArea=449
+WEBSERVER_S4L_FOGBUGZ_NEWCASE_URL=https://z43.fogbugz.com/f/cases/new?command=new&pg=pgEditBug&ixProject=45&ixArea=458
+WEBSERVER_TIS_FOGBUGZ_NEWCASE_URL=https://z43.fogbugz.com/f/cases/new?command=new&pg=pgEditBug&ixProject=45&ixArea=459
```

Comment:
minor: should we add these things in the DB?

Reply:
these are static settings for the front-end